scripts/browse.py

```diff
@@ -80,36 +82,72 @@ def create_message(chunk, question):
         "content": f"\"\"\"{chunk}\"\"\" Using the above text, please answer the following question: \"{question}\" -- if the question cannot be answered using the text, please summarize the text."
     }
 
-def summarize_text(text, question):
-    if not text:
-        return "Error: No text to summarize"
 
-    text_length = len(text)
-    print(f"Text length: {text_length} characters")
+@retry(wait=wait_exponential(multiplier=1, min=4, max=30), stop=stop_after_attempt(5))
+def call_create_chat_completion(messages):
+    return create_chat_completion(
+        model=cfg.fast_llm_model,
+        messages=messages,
+        max_tokens=300,
+
 
+def summarize_text(text, is_website=True):
+    if text == "":
```

Comment:
Why the change from `not text` to `== ""`?

Reply:
ok